model/delivery/form/LtiWizardForm.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA;
+ *
+ */
+
+namespace oat\taoLtiConsumer\model\delivery\form;
+
+use oat\generis\model\kernel\persistence\smoothsql\search\ComplexSearchService;
+use oat\taoDeliveryRdf\view\form\WizardForm;
+use oat\taoLti\models\classes\ProviderService;
+
+class LtiWizardForm extends WizardForm
+{
+    protected function initForm()
+    {
+        $this->form = new \tao_helpers_form_xhtml_Form('simpleLtiWizard');
```

Comment:
Since this is new code, can we move "old namespaces" to `use` statements insterad? Like:
```
use \tao_helpers_form_xhtml_Form as Form;
use \tao_helpers_form_FormFactory as FormFactory;
use \tao_helpers_form_xhtml_TagWrapper as TagWrapper;
...
$this->form = new Form('simpleLtiWizard');
$createElt = FormFactory::getElement('create', 'Free');
...
'actions-bottom' => new TagWrapper(['tag' => 'div', 'cssClass' => 'form-toolbar']),
```